worksheet_B/README.md

```diff
@@ -0,0 +1,56 @@
+```
+secretWord = generate a 5 letter word

```

Comment:
This only works for 5 letter words -- how could you generalise it to words of any length?